src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -35,8 +35,7 @@
     do {                                                                \
         if (sigreq)                                                     \
         {                                                               \
-            MPIDI_OFI_REQUEST_CREATE((*(sigreq)), MPIR_REQUEST_KIND__RMA); \
-            MPIR_cc_set((*(sigreq))->cc_ptr, 0);                        \
+            *(sigreq) = MPIR_Request_create_complete(MPIR_REQUEST_KIND__RMA); \
             *(flags)                    = FI_COMPLETION | FI_DELIVERY_COMPLETE; \
```

Comment:
I do not think we can safely replace this request with the lightweight one, because it is used to check per-operation completion for request-based RMA when the message is not immediately complete (see `MPIDI_OFI_INIT_CHUNK_CONTEXT`).

Instead, we can replace the "already-completed" requests such as below.
https://github.com/pmodels/mpich/blob/485ba99c84df663fc1798c246b14d1f9b1e1665c/src/mpid/ch4/netmod/ofi/ofi_rma.h#L498-L506

Reply:
Hmm, OK. I'll remove that change and do the suggested one instead. In fact, I should probably do that in a separate commit now that I look at it.

Reply:
Sounds good. Actually we can also replace the completed-requests for CH4R request-based RMA.

For example:
https://github.com/pmodels/mpich/blob/485ba99c84df663fc1798c246b14d1f9b1e1665c/src/mpid/ch4/src/ch4r_rma.h#L891-L898

Reply:
OK, I'll replace that one, too. Thanks!